src/docs/asciidoc/index.adoc

```diff
@@ -0,0 +1,13 @@
+= Speedway Trials API
+:toc:
+:toclevels: 4
+
+=== Drivers
```

Comment:
Nice setup of REST Docs, might want to use a `==` for this heading instead of `===` as it is the section after the Title. But maybe you will have a different 2nd level header before this 3rd  level header.